deploy/efs-csi/sss-config.yaml

```diff
@@ -0,0 +1,2 @@
+matchLabels:
+  hive.openshift.io/efs-csi: "true"
```

Comment:
we should also be checking for `hive.openshift.io/cluster-platform: "aws"`

Reply:
Could do, though presumably SRE wouldn't set the `efs-csi` label unless it was an AWS cluster, so is it necessary?

Reply:
In short: yes. In no case should this go onto a non-AWS cluster, so adding it as a match will not be harmful, and will prevent operator error.

Reply:
:+1: @cblecker